backend/src/govsg/middlewares/govsg-template.middleware.ts

```diff
@@ -16,20 +16,37 @@ import { CampaignService, StatsService, UploadService } from '@core/services'
 import { GovsgTemplatesAccess } from '@govsg/models'
 import { CampaignGovsgTemplate } from '@govsg/models/campaign-govsg-template'
 import { GovsgMessage } from '@govsg/models/govsg-message'
-import { GovsgTemplate } from '@govsg/models/govsg-template'
+import {
+  GovsgTemplate,
+  GovsgTemplateLanguageMetadata,
+} from '@govsg/models/govsg-template'
 import { GovsgTemplateService } from '@govsg/services'
 import { NextFunction, Request, Response } from 'express'
 
 const logger = loggerWithLabel(module)
 
+const convertMultilingualSupportToResponse = (
+  multilingualSupport: Array<GovsgTemplateLanguageMetadata>
+) => {
+  return multilingualSupport.map((languageSupport) => ({
+    body: languageSupport.body,
+    language: languageSupport.language,
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    code: languageSupport.languageCode ?? languageSupport['language_code'],
```

Comment:
I was expecting camel case keys but I got snake case keys here. I don't intend to investigate this further unless time permits.

Reply:
Back when I did this PR, I thought I was hallucinating about the cases so I added `??` as a "safety" net.

Now I know `languageSupport.languageCode` has always been `null`, I made the correction to just `languageSupport['language_code']` in one of the smaller subsequent PRs.